app/controllers/Application.scala

```diff
@@ -101,6 +101,26 @@ class Application(
     }
   }
 
+  def contributionsLandingUS(title: String, id: String, js: String, newDesigns: String): Action[AnyContent] = CachedAction() { implicit request =>
+    if (newDesigns == "circles-garnett") {
+      Ok(views.html.contributionsLanding(
+        title,
+        description = Some(stringsConfig.contributionLandingDescription),
+        "contributions-landing-page-us",
+        "contributionsLandingPageUS.js",
+        contributionsPayPalEndpoint
+      ))
+    } else {
+      Ok(views.html.contributionsLanding(
+        title,
+        description = Some(stringsConfig.contributionLandingDescription),
+        id,
+        js,
+        contributionsPayPalEndpoint
+      ))
+    }
+  }
```

Comment:
Do you think it's a good idea to reduce the duplication? I mean, both branches are the same except for `id` and `js`...

Reply:
What would be a good way to do this in Scala, ternary expression?

Reply:
Scala doesn't have ternary expressions, but perhaps something like this?
```scala
def contributionsLandingUS(title: String, id: String, js: String, newDesigns: String): Action[AnyContent] = CachedAction() { implicit request =>
  def contributionsLanding(pageId: String, pageJs: String) = Ok(views.html.contributionsLanding(
    title,
    description = Some(stringsConfig.contributionLandingDescription),
    pageId,
    pageJs,
    contributionsPayPalEndpoint
  )

  if (newDesigns == "circles-garnett")
    contributionsLanding(pageId = "contributions-landing-page-us", pageJs = "contributionsLandingPageUS.js")
  else
    contributionsLanding(id, js)
}

Reply:
I've just noticed that this is practically a duplicate of `contributionsLandingUK`. It looks like there is even more room to deduplicate them.

Reply:
Yeah it's a test across multiple pages. Ok I've had a chat with a couple people offline, I'm going to do a follow-up PR that changes all instances of this to something a bit cleaner. Thanks for the feedback! 👍